doc/sphinx/source/releasenotes.md

```diff
@@ -16,10 +16,14 @@ for each release of libCEED.
 - Promote {c:func} `CeedOperatorCheckReady`to the public API to facilitate interactive interfaces.
 - Warning added when compiling OCCA backend to alert users that this backend is experimental.
 - `ceed-backend.h`, `ceed-hash.h`, and `ceed-khash.h` removed. Users should use `ceed/backend.h`, `ceed/hash.h`, and `ceed/khash.h`.
+- Added {c:func} `CeedQFunctionGetKernelName`; refactored {c:func} `CeedQFunctionGetSourcePath` to exclude function kernel name.
 
 ### New features
 
 - `CeedScalar` can now be set as `float` or `double` at compile time.
+- Added JiT utilities in `ceed/jit-tools.h` to simplify code generation backends.
```

Comment:
I'm not sure if this first line of the new features in the release notes is completely accurate now (the purpose of the change was not really to simplify code generation backends...), so maybe we should wait until one of the later PRs to mention that?

Reply:
Sure, I can edit the message. It sort of simplifies since it collects common code from 4 places to one, but that wasn't the primary purpose.